frontend/pages/DetailedViewPage.js

```diff
@@ -40,24 +51,41 @@ export default class DetailedViewPage extends Component {
                 </View>
             );
         }
-        console.log(this.state);
+        // console.log((new Date()).getTime().getMonth());
         return (
           <View style={styles.DetailContainer}>
             <View style={styles.ItemImage}>
                 <Image
                     PlaceholderContent={<ActivityIndicator />}
-                    style={{ width: 300, height: 150 }}
+                    resizeMode='stretch'
+                    style={{flex:1, width: 150, height: 150}}
                     source={{
                         uri:
                             this.state.image_url,
                     }} />
             </View>
+            <View style={styles.HistoryContainer}>
+                <View style={styles.HistoryHeader}>
+                        <Text style={{ fontSize: 20, marginBottom: 0, paddingBottom: 0}}>History of prices at </Text>
```

Comment:
Didn't pull out styles but seems okay since they are only used once and are relatively short